diff -rupN clementine-0.2.old/src/CMakeLists.txt clementine-0.2/src/CMakeLists.txt
--- clementine-0.2.old/src/CMakeLists.txt	2010-04-17 16:55:36.000000000 -0400
+++ clementine-0.2/src/CMakeLists.txt	2010-04-17 16:55:59.000000000 -0400
@@ -198,7 +198,7 @@ include_directories("/usr/include/QxtGui
 include_directories("/usr/include/QxtCore/")
 
 set(EXECUTABLE_OUTPUT_PATH ..)
-add_library(clementine_lib
+add_library(clementine_lib STATIC
    ${CLEMENTINE-SOURCES}
    ${CLEMENTINE-SOURCES-MOC}
    ${CLEMENTINE-SOURCES-UI}
