--- clementine-0.4.2/CMakeLists.txt~	2010-07-01 21:18:22.000000000 +0300
+++ clementine-0.4.2/CMakeLists.txt	2010-07-07 09:03:25.664272595 +0300
@@ -149,12 +149,7 @@
 endif(STATIC_SQLITE)
 
 # Subdirectories
-add_subdirectory(3rdparty/qtsingleapplication)
-add_subdirectory(3rdparty/qtiocompressor)
 add_subdirectory(src)
-if (NOT APPLE)
-  add_subdirectory(3rdparty/qxt)
-endif (NOT APPLE)
 add_subdirectory(3rdparty/universalchardet)
 add_subdirectory(tests)
 add_subdirectory(dist)
--- clementine-0.4.2/src/CMakeLists.txt~	2010-07-01 21:17:36.000000000 +0300
+++ clementine-0.4.2/src/CMakeLists.txt	2010-07-07 08:33:17.521870705 +0300
@@ -5,10 +5,9 @@
 
 include_directories(${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
 include_directories(${SPARKLE})
-include_directories(../3rdparty/gmock/gtest/include)
-include_directories(../3rdparty/qtsingleapplication)
-include_directories(../3rdparty/qtiocompressor)
-include_directories(../3rdparty/qxt)
+include_directories("/usr/include/QtSolutions")
+include_directories("/usr/include/QxtGui")
+include_directories("/usr/include/QxtCore")
 include_directories(../3rdparty/libprojectm)
 
 include_directories(${GLIB_INCLUDE_DIRECTORIES})
@@ -449,8 +448,8 @@
 add_dependencies(clementine_lib pot)
 
 target_link_libraries(clementine_lib
-  qtsingleapplication
-  qtiocompressor
+  QtSolutions_SingleApplication-2.6
+  QtSolutions_IOCompressor-2.3
   chardet
   ${GOBJECT_LIBRARIES}
   ${GLIB_LIBRARIES}
@@ -475,7 +474,7 @@
   )
   include_directories(${SPARKLE}/Headers)
 else (APPLE)
-  target_link_libraries(clementine_lib qxt)
+  target_link_libraries(clementine_lib QxtCore QxtGui)
 endif (APPLE)
 
 # Link against the qsqlite plugin on windows and mac
@@ -484,7 +483,7 @@
   target_link_libraries(clementine_lib qsqlite)
 endif(HAVE_STATIC_SQLITE)
 
-add_dependencies(clementine_lib qtsingleapplication)
+add_dependencies(clementine_lib QtSolutions_SingleApplication-2.6)
 
 
 ###############################################################################
