--- clementine-0.2/dist/clementine.desktop~	2010-01-18 00:12:17.000000000 +0200
+++ clementine-0.2/dist/clementine.desktop	2010-04-24 14:09:59.250159909 +0300
@@ -6,7 +6,7 @@
 Comment=Plays music and last.fm streams
 Exec=clementine
 TryExec=clementine
-Icon=application-x-clementine
+Icon=clementine
 Terminal=false
 Categories=AudioVideo;Player;Qt;
 StartupNotify=false
--- clementine-0.5.90/dist/CMakeLists.txt~	2010-10-24 16:16:44.000000000 +0300
+++ clementine-0.5.90/dist/CMakeLists.txt	2010-12-09 00:20:22.298618631 +0200
@@ -14,8 +14,8 @@
 
 if (NOT APPLE)
   install(FILES clementine_64.png
-    DESTINATION share/icons/hicolor/64x64/apps/
-    RENAME application-x-clementine.png
+    DESTINATION share/pixmaps
+    RENAME clementine.png
   )
 
   install(FILES ../data/icon.svg
