--- clementine-0.2/dist/clementine.desktop~	2010-01-18 00:12:17.000000000 +0200
+++ clementine-0.2/dist/clementine.desktop	2010-04-24 14:09:59.250159909 +0300
@@ -6,7 +6,7 @@
 Comment=Plays music and last.fm streams
 Exec=clementine
 TryExec=clementine
-Icon=application-x-clementine
+Icon=clementine
 Terminal=false
 Categories=AudioVideo;Player;Qt;
 StartupNotify=false
--- clementine-0.4/src/CMakeLists.txt	2010-06-28 19:47:58.000000000 +0000
+++ clementine-0.4/src/CMakeLists.txt	2010-06-29 20:49:41.000000000 +0000
@@ -546,8 +546,8 @@
   )
 
   install(FILES ../dist/clementine_64.png
-    DESTINATION share/icons/hicolor/64x64/apps/
-    RENAME application-x-clementine.png
+    DESTINATION share/pixmaps
+    RENAME clementine.png
   )
 
   install(FILES ../dist/clementine.desktop
