--- clementine-0.5.3/CMakeLists.txt~	2010-10-19 14:26:26.098273180 +0300
+++ clementine-0.5.3/CMakeLists.txt	2010-10-21 15:46:12.957182252 +0300
@@ -7,7 +7,8 @@
 include(cmake/Deb.cmake)
 include(cmake/Rpm.cmake)
 
-find_package(Qt4 REQUIRED QtCore QtGui QtOpenGL QtSql QtNetwork QtXml)
+find_package(Qt4 4.5.0 REQUIRED QtCore QtGui QtOpenGL QtSql QtNetwork QtXml)
+
 if(UNIX AND NOT APPLE)
   find_package(Qt4 REQUIRED QtDbus)
 endif(UNIX AND NOT APPLE)
